framework/contrib/hit/Makefile

```diff
@@ -9,7 +9,7 @@ endif
 ifeq ($(UNAME), Darwin)
 	DYNAMIC_LOOKUP := -undefined dynamic_lookup
 else
-	DYNAMIC_LOOKUP := ""
+	DYNAMIC_LOOKUP :=
```

Comment:
Was this causing you trouble?  Because I remember explicitly adding the `""` because we had a platform that borked without it.

Reply:
TBH, that (and the change below) probably just cam from me rsyncing stuff from moosetools to moose :-D